src/main/scala/org/constellation/primitives/Schema.scala

```diff
@@ -268,7 +271,8 @@ object Schema {
     def valid: Boolean = validSrcSignature &&
       dst.address.nonEmpty &&
       dst.address.length > 30 &&
```

Comment:
Made a ticket regarding where to check addresses (instead of the direct hashes) here:

https://github.com/Constellation-Labs/constellation/issues/247